common/samples/ConsoleApp1/src/ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,4 @@
 ﻿using System;
-using System.Collections.Generic;
```

Comment:
Yeah, let's go ahead and add a sample specific to this doc then. The common samples should just stay as they are.